### PR TITLE
fix tailwind setup and other bugs

### PR DESCRIPTION
**Pull request by iryna-mertsalova**

### CODE REVIEW

File: src/styles/output.scss
Comment (IhorChai) [inline src/styles/output.scss:1]: Not sure if this file should exist in an angular project
Follow the angular guide for tailwind

Comment (IhorChai) [reply src/styles/output.scss:1]: https://tailwindcss.com/docs/guides/angular

Comment (iryna-mertsalova) [reply src/styles/output.scss:1]: I've fixed this.

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js}"],
```
Comment (IhorChai) [inline tailwind.config.js:3]: change here from {html,js} to {html,ts}

**Review by IhorChai — APPROVED** [review]

**Pull request merged**